streamz/tests/test_core.py

```diff
@@ -699,6 +699,20 @@ def test_unique_key():
     assert L == [1, 2, 3]
 
 
+def test_unique_metadata():
+    @Stream.register_api()
```

Comment:
There is already a `metdata()` function used for testing.  Can we reuse that? See the test `test_sliding_window_metadata()`.

```python
    source = Stream()
    L = metadata(source.sliding_window(2)).sink_to_list()
```

Reply:
Will do.